data/avro-schema/src/main/resources/avro/net/corda/data/ledger/utxo/contract/verification/CordaPackageSummary.avsc

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "record",
+  "name": "CordaPackageSummary",
+  "namespace": "net.corda.ledger.utxo.contract.verification",
+  "fields": [
+    {
+      "name": "name",
+      "type": "string",
+      "doc": "Name of the package."
+    },
+    {
+      "name": "version",
+      "type": "string",
+      "doc": "Version of the package."
+    },
+    {
+      "name": "signerSummaryHash",
+      "type": ["null", "string"],
```

Comment:
data/avro-schema/src/main/resources/avro/net/corda/data/packaging/CpkIdentifier.avsc
has a different type for the same field.
```
      "type": ["null", "net.corda.data.crypto.SecureHash"]
```

Reply:
This was designed to match [CordaPackageSummary](https://github.com/corda/corda-api/blob/release/os/5.0/ledger/ledger-common/src/main/kotlin/net/corda/v5/ledger/common/transaction/CordaPackageSummary.kt)